### PR TITLE
Adding LAD feature to process logs from a directory instead of single file.

### PR DESCRIPTION
**Pull request by zmhassan**
## Description:

Before this PR users would have to process logs one by one. Now they can point to a single directory and process it.

### CODE REVIEW

**Comment by zmhassan** [comment]
[Run LAD from the cli on a set of files](https://trello.com/c/Rae7LH7Y/18-run-lad-from-the-cli-on-a-set-of-files)

**Comment by zmhassan** [comment]
@durandom You can review this PR which will allow for multiple files to be processed

**Comment by zmhassan** [comment]
@4n4nd @HumairAK Please review .

File: anomaly_detector/events/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
 """Event package."""
+from anomaly_detector.events.anomaly_event import AnomalyEvent
+
+__all__ = ['AnomalyEvent']
```
Comment (HumairAK) [inline anomaly_detector/events/__init__.py]: Already updated in #225, please rebase - thanks.

File: setup.py [hunk 2]
```diff
@@ -25,7 +25,8 @@
 
 setup(
     name="scorpio",
-    version="0.0.1-rc1",
+    version="0.0.1-rc2",
```
Comment (HumairAK) [inline setup.py]: Already updated in #228 - rebase, thanks.

File: anomaly_detector/events/__init__.py (hunk 1 above)
Comment (zmhassan) [reply anomaly_detector/events/__init__.py]: That is correct. This was a cherry pick of the commits from that PR. I'm going to rebase so changes should be good now.

File: setup.py (hunk 2 above)
Comment (zmhassan) [reply setup.py]: Yes. Another cherry pick from another PR. I'll fix that. good suggestion.

**Comment by zmhassan** [comment]
@HumairAK Its all fixed. Hope you can review 2nd pass. 👍

**Pull request merged**